### PR TITLE
Update ToC for Opentelemetry Azure Monitor exporter

### PR DESCRIPTION
**Pull request by lzchen**
Also deleting old README under previous `microsoft` namespace.

### CODE REVIEW

File: docs-ref-mapping/reference-preview.yml [hunk 1]
```diff
@@ -390,10 +390,16 @@
       - azure-mgmt-mixedreality*
   - name: Monitoring
     uid: azure.python.sdk.landingPage.services.Monitor.pre
-    href: ~/docs-ref-services/preview/monitoring.md
```
Comment (scbedd) [inline docs-ref-mapping/reference-preview.yml:393]: Can you make these changes to `reference-unified.yml`? 

When you make that update, make certain to use a substitute within the path:

`href: ~/docs-ref-services/{moniker}/opentelemetry-exporter-azuremonitor-readme.md`

File: docs-ref-mapping/reference-preview.yml [hunk 2]
```diff
@@ -390,10 +390,16 @@
...
     items:
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.Monitor.Client.pre
+      href: ~/docs-ref-services/preview/opentelemetry-exporter-azuremonitor-readme.md
+      landingPageType: Service
+      children:
+      - opentelemetry-exporter-azuremonitor*
     - name: Management
       uid: azure.python.sdk.landingPage.services.Monitor.Management.pre
+      href: ~/docs-ref-services/preview/monitoring.md
```
Comment (scbedd) [inline docs-ref-mapping/reference-preview.yml:402]: Same thing as comment above.

File: docs-ref-mapping/reference-preview.yml (hunk 1 above)
Comment (lzchen) [reply docs-ref-mapping/reference-preview.yml:393]: Updated.

File: docs-ref-mapping/reference-unified.yml [hunk 3]
```diff
@@ -640,10 +640,16 @@
       - 'azure-mgmt-mixedreality*'     
   - name: Monitoring
     uid: azure.python.sdk.landingPage.services.Monitor
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:642]:
```suggestion
    uid: azure.python.sdk.landingPage.services.Monitor
    landingPageType: Service
```

File: docs-ref-mapping/reference-preview.yml [hunk 4]
```diff
@@ -390,10 +390,16 @@
       - azure-mgmt-mixedreality*
   - name: Monitoring
     uid: azure.python.sdk.landingPage.services.Monitor.pre
-    href: ~/docs-ref-services/preview/monitoring.md
     items:
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.Monitor.Client.pre
+      href: ~/docs-ref-services/preview/opentelemetry-exporter-azuremonitor-readme.md
+      landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference-preview.yml:397]:
```suggestion
```

File: docs-ref-mapping/reference-unified.yml [hunk 5]
```diff
@@ -640,10 +640,17 @@
...
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.Monitor.Client
+      href: ~/docs-ref-services/{moniker}/opentelemetry-exporter-azuremonitor-readme.md
+      landingPageType: Service
+      children:
+      - opentelemetry-exporter-azuremonitor*
     - name: Management
       uid: azure.python.sdk.landingPage.services.Monitor.Management
+      href: ~/docs-ref-services/{moniker}/monitoring.md
       landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:654]:
```suggestion
```

File: docs-ref-mapping/reference-unified.yml [hunk 6]
```diff
@@ -640,11 +640,17 @@
       - 'azure-mgmt-mixedreality*'     
   - name: Monitoring
     uid: azure.python.sdk.landingPage.services.Monitor
-    href: ~/docs-ref-services/{moniker}/monitoring.md
+    landingPageType: Service
     items:
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.Monitor.Client
+      href: ~/docs-ref-services/{moniker}/opentelemetry-exporter-azuremonitor-readme.md
+      landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:648]:
```suggestion
```

File: docs-ref-mapping/reference-preview.yml [hunk 7]
```diff
@@ -390,10 +390,15 @@
...
     items:
+    - name: Client
+      uid: azure.python.sdk.landingPage.services.Monitor.Client.pre
+      href: ~/docs-ref-services/preview/opentelemetry-exporter-azuremonitor-readme.md
+      children:
+      - opentelemetry-exporter-azuremonitor*
     - name: Management
       uid: azure.python.sdk.landingPage.services.Monitor.Management.pre
+      href: ~/docs-ref-services/preview/monitoring.md
       landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference-preview.yml:402]:
```suggestion
```

**Review by scbedd — APPROVED** [review]

**Pull request merged**